bokeh/sampledata/anscombe.py

```diff
@@ -18,6 +18,7 @@
 
 .. _Anscombe's Quartet: https://en.wikipedia.org/wiki/Anscombe%27s_quartet
 
+.. bokeh-example-sampledata:: anscombe
```

Comment:
nit: I have been trying to keep `.. _Foo` references as the very last thing in the docstrings, so this could just be moved above.

Reply:
I will make this changes later but I this is not a problem.